sqllogictest/src/runner.rs

```diff
@@ -505,14 +510,29 @@ impl<D: AsyncDB> Runner<D> {
 
     pub async fn apply_record(
         &mut self,
-        record: Record<D::ColumnType>,
-    ) -> RecordOutput<D::ColumnType> {
+        record: Record<<M::Conn as AsyncDB>::ColumnType>,
+    ) -> RecordOutput<<M::Conn as AsyncDB>::ColumnType> {
+        /// Returns whether we should skip this record, according to given `conditions`.
+        fn should_skip(
+            labels: &HashSet<String>,
+            engine_name: &str,
+            conditions: &[Condition],
+        ) -> bool {
+            conditions.iter().any(|c| {
+                c.should_skip(
+                    labels
+                        .iter()
+                        .map(|l| l.as_str())
+                        // attach the engine name to the labels
+                        .chain(Some(engine_name).filter(|n| !n.is_empty())),
```

Comment:
Why to change this in this PR? Seems not relevant 🤪

Reply:
The `engine_name` is a method of `AsyncDB`, which needs to be built from `MakeConnection` first. So we cannot have all of them in `self.labels` and call `self.should_skip` anymore. 😢